pyposeidon/utils/hfun.py

```diff
@@ -47,7 +50,7 @@ def get_hfun(dem, path=".", tag="jigsaw", resolution_min=0.01, resolution_max=0.
     cfun = fun.flatten().reshape((y.shape[0], x.shape[0]))
 
     dh = xr.Dataset(
-        {"h": (["longitude", "latitude"], cfun)},
+        {"h": (["latitude", "longitude"], cfun)},
```

Comment:
Indeed there are issues there because the `limgrad` function is a translation from `matlab` that I haven't properly reverse engineered. For the time being is a` black box`.